tests/test_iib_ops.py

```diff
@@ -293,7 +294,7 @@ def remove_operators_mock_calls_tester(
     "extra_args,push_items,mock_calls_tester",
     [
         (
-            [],
+            ["--deprecation-list", "bundle1,bundle2"],
```

Comment:
Could you add a parameter here testing when deprecation-list is passed as an empty string?

Reply:
added and change is made in source code too not only at tests